docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Add new arguments to `aql.load_file` - `native_support_kwargs` and `enable_native_fallback`. `native_support_kwargs` is a keyword argument to be used by method involved in native support flow and `enable_native_fallback` can be used to fall back to default transfer. `aql.load_file`, by default, uses native approaches to load datasets into Google BiqQuery and Snowflake. The old behaviour can be re-enabled by setting `use_native_support=False`. [#557](https://github.com/astronomer/astro-sdk/issues/557), [#481](https://github.com/astronomer/astro-sdk/issues/481)
+* Implement fallback mechanism in case native support fails to default option with log warning for problem with native support. [#557](https://github.com/astronomer/astro-sdk/issues/557)
+* Add Example DAG for Dynamic Map Task with Astro-SDK. [#377](https://github.com/astronomer/astro-sdk/issues/377),[airflow-2.3.0](https://airflow.apache.org/blog/airflow-2.3.0/)
+
+
+Internals:
+* Allow running tests on PRs from forks + label [#179](https://github.com/astronomer/astro-sdk/issues/179)
+
+
+Breaking Change:
+* Renamed `aql.truncate` to `aql.drop_table` [#554](https://github.com/astronomer/astro-sdk/issues/554)
+* Add support for customization of column titles capitalization using the argument `columns_names_capitalization=["upper", "lower", "original"]` for `aql.dataframe` and `aql.load_file` (snowflake-only) to avoid issues with Snowflake mixed-capitalized column names.[#564](https://github.com/astronomer/astro-sdk/issues/564)
+* Get configs via Airflow Configuration manager. Change the declaration for the default Astro SDK temporary schema using from `AIRFLOW__ASTRO__SQL_SCHEMA` to `AIRFLOW__ASTRO_SDK__SQL_SCHEMA` [#503](https://github.com/astronomer/astro-sdk/issues/503)
+
```

Comment:
Other breaking changes:
* `aql.load_file` attempts to load files to BigQuery and Snowflake by using native methods, which may have pre-requirements to work. To disable this mode, use XX

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/850c76559f23e121934883691e089807411cc309)